LoggingExperiences.md

```diff
@@ -0,0 +1,11 @@
+I was able to retrieve logs via the command `heroku logs`. You can also retrieve a max
+number of logs with `heroku logs -n <number>` so -> `heroku logs -n 100` to get 100 logs.
+
+I could have used `heroku logs --tail` to get a real-time display of recent logs that leaves
+the session open to get more logs.
+
+I can filter using `heroku logs -s` or `-d`. Ex, `heroku logs -s app` or `heroku logs -d router`
+
```

Comment:
Very well annotated explanation of your logging experiences. This definitely puts this project at a 3/3 starts.